src/subcommand/server/r.rs

```diff
@@ -430,6 +430,62 @@ pub(super) async fn parents(
   parents_paginated(Extension(index), Path((inscription_id, 0))).await
 }
 
+pub async fn parent_inscriptions_recursive(
```

Comment:
Since we now have a dedicated module for recursive andpoints, `_recursive` isn't needed anymore:

```suggestion
pub async fn parent_inscription(
```

Reply:
updated